guides/gatherers.md

```diff
@@ -206,13 +206,45 @@ For extra information refer to [trento-project/agent/../gatherers/packageversion
 
 ### saphostctrl
 
-This gatherer returns allows access to certain webmethods that `saphostctrl` implements. At this moment only `Ping` and `ListInstances` are supported.
+This gatherer allows access to certain webmethods that `saphostctrl` implements. An argument is required to specify
+which webmethod should be called. This webmethod is passed to the `saphostctrl` command-line tool thought the `-function` argument. Note that not all webmethods are supported.
 
-Sample arguments
-| Name                   | Return value  
-| ---------------------- | --------------------------------------------------------------------------------------------------------------------------
-| `Ping`                 | a map with two a "status" and a "elapsed" (in usec). e.g.: `{"status": "SUCCESS", "elapsed": 543341}` or `{"status": "FAILED", "elapsed": 497}`
-| `ListInstances`        | an array of maps with details of each instance, e.g.: `[{"changelist": 2069355, "hostname": "s41db", "instance": "11", "patch": 819, "sapkernel": 753, "system": "HS1"}]`
+Supported webmethods:
+  -  `Ping`
```

Comment:
It looks like an extra space got between the `-` and the word, but maybe markdown is removing it